### DISCUSSION TITLE
20240215 Weekly Development Meeting

### DISCUSSION THREAD

**Discussion by josecelano** [nautilus-cyberneering/.github#82] [Meetings]
- Time:  9:30 in Canary Islands (WET (UTC+0)).
- Time: 10:30 in Vaduz (CET (UTC+1)).

Location:
<https://meeting.blockfinance-eco.li/NautilusCyberneeringWeeklyDevMeeting>

Expected duration: 2 to 4 hours with breaks

Communicated attendants:

- CGarnham(@da2ce7)  - x
- JCelano(@josecelano) - 
- CBosse(@cgbosse) - x
- MMelchor(@mario-nt) - x
- GByrne(@grmbyrn) - x
- MVanDijke(@WarmBeer) - x

-------------------------------------------------------------

## Agenda:

1. Everyone will take their turn and:
- Go over his notes added as comments for the meeting which will have been added in advance to give an update of all the work done during the previous week indicating.
	- The issues worked on
	- The progress made on them
	- Other things done
- Inform of any faced challenges and request help if necessary to work on specific issues.
- Highlight the encountered difficulties, especially if there is a risk of them causing delays or affecting end product's quality.
- Communicate what will be the next week's tasks.
- List any new topics for further discussion in an additional Technical meeting that he noted for future discussion in comments to this discussion either before the meeting or after his turn so that they get considered.

During the meeting every team member will take their turns and follow this sequence.
The topics for further discussion will be noted in advance by each the team member and to the meetings agenda.

2. At the end of the meeting if there any new technical topics to be discussed further it will be decided to
set another meeting date for the discussion of these or to hold the meeting hereafter.

Also the persons to be part of the meeting will be decided on so that the others will be free to continue their own work.

**Comment by josecelano** [comment]
Hi @nautilus-cyberneering/core This week I will be off from Tuesday to Friday. This is what I've done since the [last meeting](https://github.com/orgs/nautilus-cyberneering/discussions/81).

### Torrust Tracker

Issues:

- [x] https://github.com/torrust/torrust-tracker/issues/691

Open new issues:

- https://github.com/torrust/torrust-tracker/issues/689

### Torrust Index

Issues:

- [x] https://github.com/torrust/torrust-index/issues/467
  - [x] https://github.com/torrust/torrust-index/issues/473
  - [x] https://github.com/torrust/torrust-index/issues/472
  - [x] https://github.com/torrust/torrust-index/issues/433
- [x] https://github.com/torrust/torrust-index/issues/468
- [x] https://github.com/torrust/torrust-index/issues/466
- [x] https://github.com/torrust/torrust-index/issues/463
- [x] https://github.com/torrust/torrust-index/issues/457
- [x] https://github.com/torrust/torrust-index/issues/424
- [x] https://github.com/torrust/torrust-index/issues/412

Pending feedback:

Please @nautilus-cyberneering/core review [my latest comment](https://github.com/torrust/torrust-index/issues/469#issuecomment-1935695015) in the issue:

https://github.com/torrust/torrust-index/issues/469#issuecomment-1935695015

That's something I would like to work on next week.

### Torrust Index GUI

Open new issues:

- https://github.com/torrust/torrust-index-gui/issues/469
- https://github.com/torrust/torrust-index-gui/issues/468

**Comment by mario-nt** [comment]
### GUI

  - https://github.com/torrust/torrust-index-gui/issues/401
  

> - GUI license page
> - Research on formatting text in vue components


### Index
- https://github.com/torrust/torrust-index/issues/445
> - https://github.com/torrust/torrust-index/pull/486	
- https://github.com/torrust/torrust-index/issues/446
> - Finished extractor and refactoring handlers
> - Worked on fixing merge with errors
 
	

### Discussions

- https://github.com/torrust/torrust-index/discussions/428

### Training/Research


- Text formatting in vue components

**Comment by mickvandijke** [comment]
I did nothing this week.
